stdlib/public/core/LifetimeManager.swift

```diff
@@ -129,9 +137,8 @@ public func _withUnprotectedUnsafeMutablePointer<T, E: Error, Result>(
 ///     `withUnsafeMutablePointer(to:_:)` instead.
 /// - Returns: The return value, if any, of the `body` closure.
 @_alwaysEmitIntoClient
-@inlinable
-public func withUnsafePointer<T, E, Result>(
-  to value: T,
+public func withUnsafePointer<T: ~Copyable, E, Result: ~Copyable>(
```

Comment:
wait why does this error type not require `: Error`?

Reply:
It's contextually known to conform to Error.

Reply:
Sure, but we're explicit in some places and implicit here? Let's be consistent 😅